stac_fastapi/pgstac/extensions/filter.py

```diff
@@ -37,5 +39,4 @@ async def get_queryables(
                 raise NotFoundError(f"Collection {collection_id} not found")
 
             queryables["$id"] = str(request.url)
-            headers = {"Content-Type": "application/schema+json"}
-            return JSONResponse(queryables, headers=headers)
+            return queryables
```

Comment:
the response class is defined here https://github.com/stac-utils/stac-fastapi/blob/main/stac_fastapi/extensions/stac_fastapi/extensions/core/filter/filter.py#L108